diff --git a/virtme/architectures.py b/virtme/architectures.py
index 6ba59e0..604eee7 100644
--- a/virtme/architectures.py
+++ b/virtme/architectures.py
@@ -77,7 +77,7 @@ class Arch_x86(Arch):
         ret = Arch.qemuargs(is_native)
 
         # Add a watchdog.  This is useful for testing.
-        ret.extend(['-watchdog', 'i6300esb'])
+        ret.extend(['-device', 'i6300esb', '-action', 'watchdog=pause'])
 
         if is_native and os.access('/dev/kvm', os.R_OK):
             # If we're likely to use KVM, request a full-featured CPU.
